src/app/(dashboard)/dashboard/FormList.tsx

```diff
@@ -0,0 +1,70 @@
+
+import React from 'react';
+import {Button} from '@/components/ui/button';
+import {Status} from '@/models/types'; // Importing the Status enum from the types file
+import type {IForm} from '@/models/types'; // Importing the Form interface from the types file
+
+
+
+
+const StatusBadge: React.FC<{status:Status}> = ({status}) => {
```

Comment:
since we already make use of clsx package (check src/lib/utils), we can make use of that to add classnames based on conditions. Doing that would prevent code duplication

Reply:
Correct, i will update it